src/state-machine/machine.ts

```
@@ -78,6 +90,31 @@ export class MeetingStateMachine {
                         return
                     }
 
+                    // Check for CAPTCHA challenges first
+                    try {
+                        const captchaResult =
+                            await this.captchaHandler.handleCAPTCHA(
+                                this.context.playwrightPage!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)